sdk/include/opentelemetry/sdk/trace/span_data.h

```diff
@@ -244,6 +259,7 @@ class SpanData final : public Recordable
   std::vector<SpanDataEvent> events_;
   std::vector<SpanDataLink> links_;
   opentelemetry::trace::SpanKind span_kind_{opentelemetry::trace::SpanKind::kInternal};
+  opentelemetry::sdk::resource::ResourceAttributes resourceAttributes_;
```

Comment:
What is this used for?  Is it worth copying the resource attributes in every span?

Reply:
Thanks for noticing it. Will make it reference/pointer to resource :)